packagedoc/additional_docs/History.tex

```diff
@@ -96,6 +96,6 @@
 - Replaced \texttt{re} package by \texttt{regex} package.}
 
 \historyversiondate{0.8.1}{5/2025}
-\historychange{Updated dependency package.}
+\historychange{regex module added to the package requirements.}
```

Comment:
Hi Son,

the proposal was:
**\texttt{**regex} module added to the package requirements

Please use the LaTeX `\texttt` command to switch the font to courier.

Reply:
Hello Holger,

Updated, sorry for my mistake because I'm not familiar with LaTeX format.